blog-en/Testing of Goroutine Leak in PouchContainer.md

```diff
@@ -0,0 +1,241 @@
+## <a name="pobkqe"></a>0. Introduction
+
+[PouchContainer](https://github.com/alibaba/pouch) is an open-source project created by Alibaba Group. It provides strong isolation and high portability, which can help enterprises quickly containerize their stock business and improve the utilization of internal physical resources. 
+
+PouchContainer is also a golang project in which goroutine has been widely used to implement management of various modules including containers, images, and logs. goroutine is an user-mode 'thread' supported by golang at the language level. This built-in feature can help developers build high-concurrency services at a rapid pace.
```

Comment:
line 5: goroutine is an user-mode 'thread' --> Goroutine is an user-mode 'thread'